service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -447,8 +451,12 @@ public String cloneWorkspace(
     List<String> applicationIds =
         applicationDao.listWorkspaceApplicationsForClone(sourceWorkspace.getWorkspaceId());
 
+    TpsPaoGetResult policy = tpsApiDispatch.getPao(sourceWorkspace.getWorkspaceId());
```

Comment:
We still have a TPS feature flag - does this call need to be behind that check? I think this is causing a unit test to fail, but I'm not sure how long we want to keep using the flag for

Reply:
I'll add it behind that check too. I'm not sure how long we want to keep using it either, but it's probably a bigger task to take it out.